azuredevops/internal/service/git/resource_git_repository_test.go

```diff
@@ -121,7 +122,7 @@ func TestGitRepo_FlattenExpand_RoundTrip(t *testing.T) {
 	require.Equal(t, *expandedGitRepo.Id, repoID)
 	require.NotNil(t, expandedProjectID)
 	require.Equal(t, *expandedProjectID, projectID)
-	require.Nil(t, repoInitialization)
+	require.Equal(t, repoInitialization.initType, string(RepoInitTypeValues.Clean))
 }
 
 func TestGitRepo_FlattenExpandInitialization_RoundTrip(t *testing.T) {
```

Comment:
Is this test testing anything different from the `TestGitRepo_FlattenExpand_RoundTrip` now? 

Maybe this? But could you move these to the above test?
```go
	require.Equal(t, repoInitialization.sourceType, "")
	require.Equal(t, repoInitialization.sourceURL, "")
```

Also, having the `configureCleanInitialization(resourceData)` come after the `flattenRepository`, seems incorrect. That wouldn't happen ever in a normal use case would it? @tmeckel

Reply:
oh yeah, they seem to be the same now; I guess I can just remove `TestGitRepo_FlattenExpand_RoundTrip` and rename `TestGitRepo_FlattenExpandInitialization_RoundTrip` as `TestGitRepo_FlattenExpand_RoundTrip`.